vlayer/ethereum/circuits/lib/src/slot.nr

```diff
@@ -13,6 +13,13 @@ pub(crate) fn mapping(slot: Bytes32, address: Address) -> Bytes32 {
     keccak256(vector.storage(), STORAGE_KEY_HASH_INPUT_LENGTH)
 }
 
+pub(crate) fn mapping_by_uint(slot: Bytes32, uint: Field) -> Bytes32 {
```

Comment:
Seems not very DRY. I would rather propose do one mapping method:
```rust
pub(crate) fn mapping(slot: Bytes32, key: Bytes32) -> Bytes32 {
  ...
}
```

Reply:
Makes sense, possible fix submitted